app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberController.scala

```diff
@@ -46,8 +46,8 @@ class EnterDuplicateMovementReferenceNumberController @Inject() (
   val sessionStore: SessionCache,
   claimService: ClaimService,
   featureSwitch: FeatureSwitchService,
-  enterDuplicateMovementReferenceNumberPage: pages.enter_duplicate_movement_reference_number,
-  enterNoLegacyDuplicateMrn: pages.enter_no_legacy_duplicate_mrn
+  enterDuplicateMovementReferenceNumberPage: pages.enter_duplicate_mrn_or_entry_number,
+  enterNoLegacyDuplicateMrn: pages.enter_duplicate_mrn
```

Comment:
this change is confusing - in the messages file there are separate sections for 

1. ENTER MRN PAGE
2. ENTER NO LEGACY MRN PAGE
3. ENTER DUPLICATE MRN PAGE
4. ENTER DUPLICATE ENTRY NUMBER PAGE
5. ENTER NO LEGACY DUPLICATE MRN PAGE

either we need a separate no legacy version of the duplicate mrn page or not - is that what you are implying here?

Reply:
I think this is in need of simplification. As you know there is a Enter MRN/ERN page and the corresponding duplicate Enter MRN/ERN page. Not sure why we have 5 different pages.

Reply:
I absolutely agree with you!
This is over complicated.
I significantly simplified it, but only because I had to.
The error I was fixing was in the EnterMovementReferenceNumberController, but the to controllers were coupled, 
so I had to change the EnterDuplicateMovementReferenceNumberController  too.
Refactoring and cleaning up the EnterDuplicateMovementReferenceNumberController  was not part of this ticket, 
therefore - unless you insist - I will not do it.
Please create a separate ticket for cleaning up the duplicate journey.
At the same time, I did change the language keys and templates, to better reflect my re-naming, in my next commit.